pkg/cmd/profiledata/env/env.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2024 Daytona Platforms Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package env
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var EnvCmd = &cobra.Command{
```

Comment:
Since we switched to "everything is server" logic in the CLI and removed things such as provider and target from the `server` subcommand, we should discuss if this command would then belong to the `profile` subcommand @vedranjukic

Reply:
I see what you're getting at, but env vars are actually handled by the server as well so it kind of falls in the same bucket as provider and target commands.

Reply:
The only thing I would add is a note that these are "global" env variables (that will apply to all new workspaces)

Reply:
`Manage profile environment variables that are added to all workspaces`

This is the description of the command.